prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -272,7 +284,15 @@ class Hl7Serializer(val metadata: Metadata) {
         val suppressQst = hl7Config?.suppressQstForAoe ?: false
         val suppressAoe = hl7Config?.suppressAoe ?: false
         // and we have some fields to suppress
-        val suppressedFields = hl7Config?.suppressHl7Fields?.split(",") ?: emptyList()
+        val suppressedFields = hl7Config
+            ?.suppressHl7Fields
+            ?.split(",")
+            ?.map { it.trim() } ?: emptyList()
+        // or maybe we're going to suppress UNK/ASKU for some fields
```

Comment:
Trim up the suppressed fields so we don't miss matching if someone has spaces after commas in the list.  This does not get lower cased like report element names below because the HAPI parser does not know what to do with `obx` vs `OBX`. 🤷‍♂️